x-pack/platform/plugins/shared/saved_objects_tagging/public/components/edition_modal/create_or_edit_modal.tsx

```diff
@@ -112,9 +113,14 @@ export const CreateOrEditModal: FC<CreateOrEditModalProps> = ({
   );
 
   return (
-    <EuiModal onClose={onClose} initialFocus="[name=name]" style={{ minWidth: '600px' }}>
+    <EuiModal
+      onClose={onClose}
+      initialFocus="[name=name]"
```

Comment:
Could you please also remove initialFocus="[name=name]" . Initial focus should be on dialog, not on name